### ISSUE TITLE
events: panic when calling method `User.Events`

### ISSUE THREAD

**Issue by zapateo** [krenalis/krenalis#403]
When calling the method `User.Events` and the user has some events associated to him, Chichi panics with the error:

```
interface conversion: interface {} is *errors.errorString, not string
```

at `chichi/apis/users.go:413`.

Anyway, the PR https://github.com/open2b/chichi/pull/399 changes the code of `User.Events`, so consider this issue after mergin that PR, if this still applies.

**Comment by zapateo** [comment]
There are two different problems:

1. The error messages in the `normalize` functions of the data warehouses mistakenly returns the error as a value, instead of returning it as an error
2. The `traits` property is declared as an Object, but it is stored in the data warehouse as a JSON/text, which seems to be not handled by the `normalize`

Fixing just the first problem makes Chichi fail every time that an event is read; currently, instead, the error is silently handled as a string value.

**Comment by zapateo** [comment]
Since both the problems above have been solved (commits 3f4a9079834adc35270ea1001984f48bf90c6f14 and 1e713b37a5535d36f20bd1f2e047dc80d7b253a2), this issue can be closed.
